package.json

```diff
@@ -58,12 +58,12 @@
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.28.13",
+    "ember-data": "~4.6.0",
     "ember-load-initializers": "^2.1.2",
     "ember-page-title": "^7.0.0",
     "ember-qunit": "^6.1.1",
     "ember-resolver": "^10.0.0",
-    "ember-source": "~3.28.12",
+    "ember-source": "~4.6.0",
```

Comment:
@dwickern did you mean to keep these at 4.6?

Reply:
Yes, so that we're testing against 3.28 / 4.4 / 4.6

Reply:
Should we add ember-data as a peer dependency?

Reply:
Not sure, is that recommended now @runspired ?